investment-analysis-on-edgar-using-vss/README.md

```diff
@@ -0,0 +1,51 @@
+# Investment Analysis using Vector Similarity in Amazon Neptune Analytics - Demo
+
+This repository contains a practical demo showcasing the concepts discussed in the "Investment Analysis using Vector Similarity in Amazon Neptune Analytics" blog series. The demo covers setting up a data pipeline to convert the raw EDGAR dataset to a Knowledge Graph, running Vector Similarity Search (VSS) combined with graph traversal queries, and provides a frontend for visualization.
+
+## Preprocessed Knowledge Graph
+
+For your convenience, we have already prepared the processed and ready-to-use EDGAR Knowledge Graph, which is stored at:
```

Comment:
Please also detail: 1/ where to get the original data, 2/ how we sourced the descriptions, 3/ how we created the embeddings, and 4/ include a header of the first few lines to show the format.